ci-runner/vendor/github.com/devtron-labs/common-lib/imageScan/bean/executionBean.go

```diff
@@ -0,0 +1,16 @@
+package bean
+
+type ExecutionMedium string
+
+const (
+	Rest  ExecutionMedium = "rest"
+	Steps ExecutionMedium = "steps"
```

Comment:
can we rename this to somethink more in line with inHouse image scanner vs external?